source/manual/jenkins-ci.html.md

```diff
@@ -133,6 +131,82 @@ With this configuration, in our jobs we can choose between 'GitHub' and 'GDS Git
 access the source code repository. The API endpoint name is referenced in Puppet to configure jobs automatically so
 we shouldn't update it.
 
+## Patching
```

Comment:
Would it make sense to create a separate page for this so it can be found more easily?